cellmlmanip/model.py

```diff
@@ -387,6 +387,20 @@ def get_derived_quantities(self):
                               and node.get('variable_type', '') not in ('state', 'free', 'parameter')]
         return sorted(derived_quantities, key=lambda var: var.order_added)
 
+    def get_display_name(self, var, META='https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata#'):
+        """Return a display name for the given variable.
+
+        Looks for META ontology annotation tag first, then cmeta:id if present, or the name attribute if not.
+        If there is an interface component, strip the name of it out of the display name.
+        :param var: the variable for which to get the display name.
+        :param META: the namespace prefix for the ontology used, (by default the oxford chaste metadata ontology).
+
+        :return: ontology annotation if the var has it, otherwise cmeta_id, if it has it otherwise variable name.
```

Comment:
Perhaps "the local name from the preferred ontology (if such an annotation exists), or the cmeta_id (if present) or the variable's name"